examples/jsm/loaders/3DMLoader.js

```diff
@@ -418,7 +406,7 @@ class Rhino3dmLoader extends Loader {
 					const xf = iRef.geometry.xform.array;
 
 					const matrix = new Matrix4();
-          			matrix.set( xf[ 0 ], xf[ 1 ], xf[ 2 ], xf[ 3 ], xf[ 4 ], xf[ 5 ], xf[ 6 ], xf[ 7 ], xf[ 8 ], xf[ 9 ], xf[ 10 ], xf[ 11 ], xf[ 12 ], xf[ 13 ], xf[ 14 ], xf[ 15 ] );
+					matrix.elements = xf;
```

Comment:
Please avoid to change property references like that. If some code has a reference on the existing `elements` array, assigning a new array elsewhere will break it.

Reply:
Fixed!